src/Aspire.Hosting/MongoDB/MongoDBBuilderExtensions.cs

```diff
@@ -73,6 +73,41 @@ public static IResourceBuilder<MongoDBDatabaseResource> AddDatabase(this IResour
         return builder;
     }
 
+    /// <summary>
+    /// Adds a named volume for the data folder to a MongoDb container resource.
+    /// </summary>
+    /// <param name="builder">The resource builder.</param>
+    /// <param name="name">The name of the mount. Defaults to an auto-generated volume. </param>
+    /// <param name="isReadOnly">A flag that indicates if this is a read-only mount.</param>
+    /// <returns>The <see cref="IResourceBuilder{T}"/>.</returns>
+    public static IResourceBuilder<MongoDBServerResource> WithDataVolume(this IResourceBuilder<MongoDBServerResource> builder, string? name = null, bool isReadOnly = false)
+    {
+        name ??= $".data/{builder.Resource.Name}";
```

Comment:
When auto-naming volumes (rather than directories) I think we should use a form more like `"${builder.Resource.Name}-data"`.

Reply:
I know, this is already fixed (the second commit i pushed today). The pipeline for the fix failed on the "Run helix tests" task, doesn't seem like a problem with my changes (https://dev.azure.com/dnceng-public/public/_build/results?buildId=596222&view=logs&jobId=0bc77094-9fcd-5c38-f6e4-27d2ae131589&j=0bc77094-9fcd-5c38-f6e4-27d2ae131589&t=06901622-1165-5cb8-9fdb-a339481d3deb)